tests/decrypt-secrets.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+travis encrypt-file ${DIR}/secrets.tar.enc --decrypt --key $TRAVIS_CI_KEY --iv $TRAVIS_CI_IV
```

Comment:
what's `IV` in the environment variable stand for?

Reply:
Part of openssl. I couldn't find in man page but googled it: https://stackoverflow.com/a/39412887. According to unofficial sources it's Initialization Vector which is part of the AES encryption.